doc/performance-investigations/extract_distribution_summary.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+
+get_values = []
+post_values = []
+total = 0
+
+for line in sys.stdin:
+    if 'Name' in line:
+        continue
+
+    values = line.split(',')
+    percentile_99th = int(values[9])
+
+    if 'GET /questionnaire' in line:
+        get_values.append(percentile_99th)
+    if 'POST /questionnaire' in line:
+        post_values.append(percentile_99th)
+    if 'Total' in line:
```

Comment:
Total is not valid?
```suggestion
    if 'Aggregated' in line:
```

Reply:
I've just checked in a clean virtual env and my file has "Total". 

On the repeating of tests, what do you suggest? I'm aware that if we need to run lots of tests it will slow down progress.

Reply:
So as discussed, the latest version of locust seems to use `Aggregated` so could be worth upgrading but will leave it up to you.

As for the repeating of the tests, I think it worthwhile to at least test the changes twice to potentially reduce the chance that it is an anomaly?. I guess it could be that the tests are run 2-3 times but we only check-in 1 of the results on the basis that they are similar.

Reply:
I've added a line about running the tests more than once. I've opened a separate pr for updating the Python modules as it didn't seem related to a pull request adding documentation.